client/src/app/pages/reports/reports.tsx

```diff
@@ -78,13 +129,54 @@ export const Reports: React.FC = () => {
               <StackItem>
                 <Card>
                   <CardHeader>
-                    <TextContent>
-                      <Text component="h3">{t("terms.currentLandscape")}</Text>
-                    </TextContent>
+                    <Flex>
+                      <FlexItem>
+                        <TextContent>
+                          <Text component="h3">
+                            {t("terms.currentLandscape")}
+                          </Text>
+                        </TextContent>
+                      </FlexItem>
+                      <FlexItem>
+                        <Select
+                          id="select-questionnaires"
+                          isOpen={isQuestionnaireSelectOpen}
+                          selected={selectedQuestionnaire}
+                          onSelect={onSelectQuestionnaire}
+                          onOpenChange={(_isOpen) =>
+                            setIsQuestionnaireSelectOpen(false)
+                          }
+                          toggle={toggleQuestionnaire}
+                          shouldFocusToggleOnSelect
+                        >
+                          <SelectOption key={0} value="All questionnaires">
```

Comment:
I suggest using a constant value here to make sure it is the same when you need to test it lower...

```jsx
  const ALL_SELECTED = "all-questionnaires";
  ...

  value={ALL_SELECTED}
```

Reply:
@sjd78, +1 for using a const meanwhile with a name value (please read below for explanation).